src/api/shared/create-redis.ts

```diff
@@ -17,7 +20,13 @@ export function createRedisFactory(redisUrl?: string): () => IORedis.Redis {
   let redis: IORedis.Redis | undefined = undefined;
   return () => {
     if (!redis) {
-      redis = new IORedis(redisUrl);
+      const options: RedisOptions = {}
+      if (process.env.REDIS_TLS_CA_FILE && fs.existsSync(process.env.REDIS_TLS_CA_FILE)) {
```

Comment:
this handles a non-existing REDIS_TLS_CA_FILE silently - the alternative would be to throw an error if the file doesn't exist. What do you prefer?

Reply:
From my own experience of running against a redis requiring tls without providing one: ioredis will complain anyway. We just would have the opportunity to throw a nicer error probably.